documentation/bolt_known_issues.md

```diff
@@ -1,39 +1,65 @@
 # Known issues
 
-Known issues for the Bolt 1.x release series.
-
 ## Tasks executed with PowerShell version 2.x or earlier cannot use parameters named `type`
 
-When executing PowerShell tasks on targets using a PowerShell interpreter
-version 2.x or earlier, you cannot use a task parameter with the name `type`.
+When executing PowerShell tasks on targets using PowerShell version 2.x or
+earlier, you cannot use a task parameter with the name `type`. Since PowerShell
+version 2.x and earlier do not support `type` as a named argument, and
+PowerShell tasks convert parameters to named arguments, a `type` parameter will
+be filtered out before running the task.
```

Comment:
```suggestion
When executing PowerShell tasks on targets using PowerShell version 2.x or
earlier, you cannot use a task parameter with the name `type`. Because PowerShell
version 2.x and earlier do not support `type` as a named argument, and
PowerShell tasks convert parameters to named arguments, Bolt will filter out a `type` 
parameter before running the task.
```
Style tip: "Since" is fine grammatically, but we tend to avoid it because it's ambiguous for non native speakers.

Reply:
Good to know!